crux_core/Cargo.toml

```diff
@@ -41,3 +41,4 @@ serde = { version = "1.0.201", features = ["derive"] }
 static_assertions = "1.1"
 rand = "0.8"
 url = "2.5.0"
+uuid = { version = "1.8.0", features = ["v4", "serde"] }
```

Comment:
Is it still needed for dev...? 👀

Reply:
Not really. There's a test for sending UUID across the typegen IIRC. Could use another type though